CONTRIBUTING.md

```diff
@@ -44,6 +40,13 @@ Install dependencies by running:
 pnpm i
```

Comment:
Since you're updating this file, maybe we could avoid shortcuts in the commands. I tried fixing similar shortcuts in css and drive repos.
```suggestion
pnpm install
```